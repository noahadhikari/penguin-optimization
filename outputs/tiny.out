# Penalty = 510
3
8 1
2 2
6 8
