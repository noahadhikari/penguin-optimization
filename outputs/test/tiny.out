# Penalty = 510
3
2 2
8 1
6 8
